troubleshooting/tools/log4j-tcp-app/src/main/java/com/mycompany/app/App.java

```diff
@@ -0,0 +1,129 @@
+package com.mycompany.app;
+
+//import org.apache.log4j.Logger;
```

Comment:
There are some codes commented in this file. What are they for? Should we delete them?